javascript/javascript1/week3/travelInfo.js

```diff
@@ -0,0 +1,8 @@
+function timeToTravel(speed, distance)
+    {
+        var time=distance/speed;
+        return time;
+    }
+
+var myTime=timeToTravel(50,100);
```

Comment:
"The time should be formatted like this: 3 hours and 34 minutes" You didn't finish the exercise

Reply:
I changed the code here...
https://github.com/Aswini-D3/hyf-homework/blob/javascript/javascript1/week3/javascript/javascript1/week3/travelInfo.js
please check.